File: Py (3)/11.py
def intercambiar(valores):
    valores[0], valores[1] = valores[1], valores[0]

# Programa principal
x = int(input("Ingrese primer valor: "))
y = int(input("Ingrese segundo valor: "))

valores = [x, y]
intercambiar(valores)

print("Después del intercambio: x =", valores[0], "y =", valores[1])

File: C++(3)/11.cpp
#include <iostream>
using namespace std;

void intercambiar(int &a, int &b) {
    int temp = a;
    a = b;
    b = temp;
}

int main() {
    int x, y;
    cout << "Ingrese primer valor: ";
    cin >> x;
    cout << "Ingrese segundo valor: ";
    cin >> y;

    intercambiar(x, y);

    cout << "Después del intercambio: x = " << x << ", y = " << y << endl;
    return 0;
}
